frontend/src/pages/Contact.jsx

```
@@ -0,0 +1,368 @@
+                    <h2 className="text-3xl font-bold mb-6 text-gray-800">Join Our Network</h2>
+                    <p className="text-lg text-gray-700 mb-8">
+                      Are you a property owner looking to partner with an exclusive platform? Join our network of elite accommodation providers and expand your reach.
+                    </p>
+                    
+                    <button 
+                      onClick={() => {
+                        setActiveSection('contact');
+                        setTimeout(() => setShowVendorForm(true), 300);
+                      }}
+                      className="bg-blue-600 text-white py-4 px-8 rounded-full hover:bg-blue-700 transition duration-300 flex items-center text-lg font-medium shadow-lg hover:shadow-xl cursor-pointer"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)